src/ir.cpp

```diff
@@ -4448,6 +4463,8 @@ static IrInstruction *ir_gen_builtin_fn_call(IrBuilder *irb, Scope *scope, AstNo
             return ir_lval_wrap(irb, scope, ir_build_return_address(irb, scope, node), lval);
         case BuiltinFnIdFrameAddress:
             return ir_lval_wrap(irb, scope, ir_build_frame_address(irb, scope, node), lval);
+        case BuiltinFnIdHandle:
+            return ir_lval_wrap(irb, scope, ir_build_handle(irb, scope, node), lval);
```

Comment:
This is probably actually the best place to put the compile error for `@handle()` used outside an async function. Search for `return expression outside function definition` to see an example.